package.json

```diff
@@ -105,8 +105,8 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@alfresco/adf-cli": "4.2.0",
```

Comment:
those two dep should actually not be here

Reply:
In the develop branch they are present. Should they not even there, should I remove them completely in my PR?